### DISCUSSION TITLE
Skin Backup

### DISCUSSION THREAD

**Discussion by DROIDshield** [jurialmunkey/skin.arctic.horizon.2#914] [Discussion]
@jurialmunkey hi, I know this question has been asked before but nothing I think came of it so I'm raising again so I can find out for myself.

Will you be implementing a skin backup at all or anytime soon?

My skin is setup now so complex and I absolutely love it. But the only way I can backup the complete system is to do a backup in Kodi and copy the org.xbmc.kodi file. Now this works....but its a massive file well mine is some 5.5GB this takes a long time to populate on a sheild with only 3gig of ram to run.

My settings in Kodi ie passthrough and slight tweaks here and there are easy to put back if it failed or crashed.

If AH2 fails or crashes I'm going to sit, hold my head and cry...there are just too many tweaks, widgets in AH2 to remember an what i have also done with backgrounds etc to make this skin truly unique to get it back to pre crash would be a killer if it does stop working as with most any android apk skin etc, they are prone to crash at some point or other.

So for the moment I will again backup and take the large slow file from Kodi because its been a while since I did this and you have made many updates since.
It would be fantastic to have the skin backup.

Let me know what your thoughts are on this matter.

Thanks

**Comment by jurialmunkey** [comment]
No need to backup such a huge file, all you need are the following folders from userdata/addon_data/

skin.arctic.horizon.2
script.skinshortcuts
script.skinvariables

That will cover skin settings, skinshortcuts setup and viewtypes

Optionally you could backup the settings.xml for tmdbhelper (I wouldn't bother with the rest of the contents of its addon data folder as it will be mostly caches which will take up lots of space).

**Reply by DROIDshield** [reply]
Oh...man thank you. That will help.

I actually left my shield backing up lastnight before bed and it was 6.54 gig🤔 checked it this morning before work and it had failed.. 😡

So I will try you way later. This will be a massive help. Thanks again.

**Reply by xyzfre** [reply]
If you use clear logos usually before I do a back up  I go into user/addon_data TMDb Helper and delete the folders that say Crop and database_v6 this helps make the back up folder smaller

**Reply by DROIDshield** [reply]
@xyzfre I saw the folder... are these folders that need to be cleared anyway from time to time to keep the system running smoothly? I guess on my shields with limited ram maybe?
But in a pc like my laptop with 32gig ram no need.
![20230421_190901](https://user-images.githubusercontent.com/97023369/233757919-aed1e4d7-7866-42e2-9020-d37743710e40.jpg)
